pwa/app/components/tba/match/breakers.tsx

```diff
@@ -0,0 +1,119 @@
+import { Event, Match } from '~/api/tba/read';
+import { DOUBLE_ELIM_ROUND_MAPPING, PlayoffType } from '~/lib/api/PlayoffType';
+import { COMP_LEVEL_SHORT_STRINGS } from '~/lib/matchUtils';
+import { timestampsAreOnDifferentDays } from '~/lib/utils';
+
+export interface BreakerResult {
+  shouldBreak: boolean;
+  text?: string;
+  whereToInsertBreak?: 'before' | 'after';
+}
+
+export type ShouldInsertBreakCallback = (
+  match: Match,
```

Comment:
can this be done with destructuring instead of positional arguments and eliminate the problem with unused vars?